spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task) }
+  let(:task) { create(:task, :done) }
```

Comment:
`let`で定義しているキーがどちらも`:task`となっているので、5行目と6行目どちらのFactory作成が行われるか分からないですね。

`:done`の`trait`がある方はキーの名前も変えておきましょう！
```suggestion
  let(:task) { create(:task) }
  let(:done_task) { create(:task, :done) }
```